sdk/eventhub/azure-eventhub/tests/perfstress_tests/README.md

```diff
@@ -1,72 +1,90 @@
-# EventHub Performance Tests
+#  Event Hubs Python client performance tests
 
-In order to run the performance tests, the `azure-devtools` package must be installed. This is done as part of the `dev_requirements`.
-Start by creating a new virtual environment for your perf tests. This will need to be a Python 3 environment, preferably >=3.7.
-Note that tests for T1 and T2 SDKs cannot be run from the same environment, and will need to be setup separately.
+In order to run the performance tests, the `azure-devtools` package must be installed. This is done as part of the `dev_requirements.txt`. installation. Start by creating a new virtual environment for your perf tests. This will need to be a Python 3 environment, preferably >=3.7.
 
 ### Setup for test resources
 
-These tests will run against a pre-configured EventHub. The following environment variable will need to be set for the tests to access the live resources:
+These tests will run against a pre-configured EventHub. The following environment variables will need to be set for the tests to access the live resources:
+
 ```
 AZURE_EVENTHUB_CONNECTION_STRING=<the connection string of an Event Hub.>
 AZURE_EVENTHUB_NAME=<the path of the specific Event Hub to connect to>
 ```
 
-### Setup for T2 perf test runs
+If using Azure Blob Storage for checkpointing, the following environment variable will need to be set:
 
-```cmd
-(env) ~/azure-eventhub> pip install -r dev_requirements.txt
-(env) ~/azure-eventhub> pip install .
+```
+AZURE_STORAGE_CONNECTION_STRING=<the connection string of an Azure Storage account>
 ```
 
-### Setup for T1 perf test runs
+### Setup for perf test runs
```

Comment:
just an FYI, we are removing mentions of T1 & T2 ?

Reply:
T1 legacy tests were removed a while back, so thought it wouldn't make sense to still mention T1 and T2.